markets/perps-market/contracts/modules/GlobalPerpsMarketModule.sol

```diff
@@ -246,4 +248,53 @@ contract GlobalPerpsMarketModule is IGlobalPerpsMarketModule {
         maxPositionsPerAccount = store.maxPositionsPerAccount;
         maxCollateralsPerAccount = store.maxCollateralsPerAccount;
     }
+
+    /**
+     * @inheritdoc IGlobalPerpsMarketModule
+     */
+    function setInterestRateParameters(
+        uint128 lowUtilizationInterestRateGradient,
+        uint128 interestRateGradientBreakpoint,
+        uint128 highUtilizationInterestRateGradient
+    ) external override {
+        OwnableStorage.onlyOwner();
+        GlobalPerpsMarketConfiguration.Data storage store = GlobalPerpsMarketConfiguration.load();
+        store.lowUtilizationInterestRateGradient = lowUtilizationInterestRateGradient;
+        store.interestRateGradientBreakpoint = interestRateGradientBreakpoint;
+        store.highUtilizationInterestRateGradient = highUtilizationInterestRateGradient;
```

Comment:
should we check that `highUtilizationInterestRateGradient >= lowUtilizationInterestRateGradient` as a checkpoing? I don't find a reasonable usage of `highUtilizationInterestRateGradient` being low that `lowUtilizationInterestRateGradient`

Reply:
i'd say nice to have but will add it.  if the gradient is lower, then it will always just use whatever the high gradient value is so codewise it won't break but probably good to have some guardrails 👍🏽